internal/go/skycfg/proto_enum.go

```diff
@@ -74,6 +77,24 @@ func (v *skyProtoEnumValue) Hash() (uint32, error) {
 	return starlark.MakeInt64(int64(v.value)).Hash()
 }
 
+func (v *skyProtoEnumValue) CompareSameType(op syntax.Token, y starlark.Value, depth int) (bool, error) {
+	other, ok := y.(*skyProtoEnumValue)
+	if !ok {
+		return false, fmt.Errorf("`%v' not enum value", y)
+	}
+
+	eq := (v.typeName == other.typeName) && (v.valueName == other.valueName) && (v.value == other.value)
```

Comment:
This is going to return not-equal for enum values with the same integer code but different names, which is different from how `Hash()` works -- I believe that would cause problems for code that uses enum values as dict keys.

Reply:
Good point. Fixed